### PR TITLE
Fix ETL scripts

### PR DESCRIPTION
**Pull request by calbach**
Currently must be run from `rest-api`. Fix `--noclone` export flags.

### CODE REVIEW

File: rest-api/etl/export_etl_results.sh [hunk 1]
```diff
@@ -22,7 +22,10 @@ fi
 PROJECT_AND_ACCOUNT=
 if [ "${PROJECT}" ]
 then
-  PROJECT_AND_ACCOUNT="--project ${PROJECT} --account ${ACCOUNT} --instance_name ${INSTANCE_NAME} --service_account exporter@${PROJECT}.iam.gserviceaccount.com"
+  PROJECT_AND_ACCOUNT="--project ${PROJECT} --account ${ACCOUNT} --service_account exporter@${PROJECT}.iam.gserviceaccount.com"
+fi
+if [ -n "${INSTANCE_NAME}" ]; then
```
Comment (danqrodney) [inline rest-api/etl/export_etl_results.sh]: This is fine but you'd never use INSTANCE_NAME without also using PROJECT?

Comment (danqrodney) [reply rest-api/etl/export_etl_results.sh]: (So I guess this could be inside the former if statement)

Comment (calbach) [reply rest-api/etl/export_etl_results.sh]: Done.

What does it actually mean to omit project/account. Does that send against local?

Comment (danqrodney) [reply rest-api/etl/export_etl_results.sh]: Yep.

**Pull request merged**